### PR TITLE
Import duplicating tasks issue

### PR DESCRIPTION
**Pull request by mrgambal**
### Shiny improvements were brought by code-fairies ###

- [x] no more silly overwrirtings
- [x] repository doesn't know anything 'bout web-nature of the project (got rid of 404 exceptions)
- [x] tests were extended a bit

### CODE REVIEW

File: vulyk/models/task_types.py [hunk 1]
```diff
@@ -75,16 +76,20 @@ def import_tasks(self, tasks, batch):
...
+                assert isinstance(task, dict)
+
+                bulk.append(self.task_model(
                     id=sha1(json.dumps(task).encode('utf8')).hexdigest()[:20],
                     batch=batch,
                     task_type=self.type_name,
-                    task_data=task,
-                )
+                    task_data=task))
+            self.task_model.objects.insert(bulk, load_bulk=True)
```
Comment (dchaplinsky) [inline vulyk/models/task_types.py]: What will happen if such task already exists? Whole bulk import will die and transaction will be rolled back?

Comment (mrgambal) [inline vulyk/models/task_types.py]: Let's write a test for that!

Comment (mrgambal) [inline vulyk/models/task_types.py]: Unforttunately - it won't.
It will store all tasks preceding to the one caused exception.

**Comment by mrgambal** [comment]
@dchaplinsky check that one

**Pull request merged**